napari/layers/image/_tests/test_image_utils.py

```diff
@@ -86,7 +86,7 @@ def test_guess_multiscale_non_array_list():
     """Check that non-decreasing list input raises ValueError"""
     data = [
         np.empty((10, 15, 6)),
-    ] * 2  # noqa: E231
+    ] * 2
```

Comment:
We can probably but this bac in 1 line; does this not happen automatically with black?

Reply:
black do not reformat it because of s colon after element.